helm/docs-indexer-app/templates/cronjob-blog.yaml

```diff
@@ -50,8 +50,10 @@ spec:
                 seccompProfile:
                   type: RuntimeDefault
               env:
+                - name: OPENSEARCH_ENDPOINT
+                  value: {{ .Values.opensearchEndpoint | default .Values.elasticsearchEndpoint }}
                 - name: ELASTICSEARCH_ENDPOINT
-                  value: {{ .Values.elasticsearchEndpoint }}
+                  value: {{ .Values.opensearchEndpoint | default .Values.elasticsearchEndpoint }}
```

Comment:
Is this a mistake? ELASTICSEARCH_ENDPOINT set to .Values.opensearchEndpoint ?

Reply:
good catch